outlines/text/generate/continuation.py

```diff
@@ -88,10 +91,16 @@ def postprocess_completions(self, completions: List[str]) -> List[str]:
         return completions_without_stop
 
 
-def continuation(
+@functools.singledispatch
+def continuation(model, max_tokens):
```

Comment:
I believe `pytest` fails because this function needs to have the same signature as the other ones.

Reply:
I don't think so. Looks like the `Model()` class defined in the test doesn't match the types we have created in this dispatch. So it throws a `NotImplementedError``